plugin/src/main/java/com/denizenscript/denizen/objects/properties/entity/EntityTemper.java

```diff
@@ -0,0 +1,52 @@
+package com.denizenscript.denizen.objects.properties.entity;
+
+import com.denizenscript.denizen.objects.EntityTag;
+import com.denizenscript.denizencore.objects.Mechanism;
+import com.denizenscript.denizencore.objects.core.ElementTag;
+import org.bukkit.entity.AbstractHorse;
+
+public class EntityTemper extends EntityProperty<ElementTag> {
+
+    // <--[property]
+    // @object EntityTag
+    // @name temper
+    // @input ElementTag(Number)
+    // @description
+    // Controls how close to being fully tamed a horse-type entity is.
+    // A value of "0" indicates that no action has been done to try and domesticate this entity.
+    //
+    // When a player is riding the entity, a random number between 0 and 99 is chosen.
+    // The entity becomes tame if this number is less than the temper, otherwise, the temper is
+    // increased by 5 and the player is bucked off. Temper can also be increased by feeding the entity.
```

Comment:
don't do newline splits in the middle of a sentence like this

Reply:
Also, what is the initial temper value for vanilla horses and all?

And how does max_temper factor in? The description of max_temper and temper properties imply two entirely different taming systems